src/main/java/com/project/bookstore/model/Book.java

```diff
@@ -7,10 +7,8 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
-import lombok.Data;
 
 @Entity
-@Data
```

Comment:
use Getter and Setter lombok annotations here
use Data for dto classes
why have you removed lombok?

Reply:
i had problem with lombok, it wasn`t working with mapstruct, it wasn`t even compiling. And annotation @Data and @Getter and @Setter also weren`t working.  In process of compiling it was making an error that was caused by absent of getter

Reply:
this could be fixed. just google.